katello/katello-restore

```diff
@@ -1,24 +1,30 @@
 #!/usr/bin/env ruby
 
+require "socket"
 require 'optparse'
 require 'fileutils'
 require 'date'
 
-STANDARD_BACKUP_FILES = ['config_files.tar.gz', 'pulp_data.tar']
+STANDARD_BACKUP_FILES = ['config_files.tar.gz', 'pulp_data.tar*']
```

Comment:
I put `---- *pulp_data.tar is optional ----` [here](https://git.io/vHMKT) as its my understanding that you don't need pulp_data.tar to restore? Please correct me if I'm off on this.